apps/betterangels-backend/shelters/enums.py

```diff
@@ -4,196 +4,206 @@
-    ROADMAP_HOME = "Roadmap Home", _("Roadmap Home")
-    SAFE_PARK_LA = "Safe Park LA", _("Safe Park LA")
-    SOBER_LIVING = "Sober Living", _("Sober Living")
-    TINY_HOME_VILLAGE = "Tiny Home Village", _("Tiny Home Village")
-    TRANSITIONAL_HOUSING = "Transitional Housing", _("Transitional Housing")
-    WINTER_SHELTER = "Winter Shelter", _("Winter Shelter")
+    A_BRIDGE_HOME = "a_bridge_home", _("A Bridge Home")
+    CRISIS_HOUSING = "crisis_housing", _("Crisis Housing")
+    EMERGENCY_SHELTER = "emergency_shelter", _("Emergency Shelter")
+    FAITH_BASED = "faith_based", _("Faith Based")
+    INTERIM_HOUSING = "interim_housing", _("Interim Housing")
+    PERMANENT_HOUSING = "permanent_housing", _("Permanent Housing")
+    PROJECT_HOMEKEY = "project_homekey", _("Project Home Key (PHK)")
+    RAPID_REHOUSING = "rapid_rehousing", _("Rapid Rehousing")
+    RECUPERATIVE_CARE = "recuperative_care", _("Recuperative Care")
+    ROADMAP_HOME = "roadmap_home", _("Roadmap Home")
+    SAFE_PARK_LA = "safe_park_la", _("Safe Park LA")
+    SOBER_LIVING = "sober_living", _("Sober Living")
+    TINY_HOME_VILLAGE = "tiny_home_village", _("Tiny Home Village")
+    TRANSITIONAL_HOUSING = "transitional_housing", _("Transitional Housing")
+    WINTER_SHELTER = "winter_shelter", _("Winter Shelter")
 
 
 class PopulationChoices(models.TextChoices):
-    ADULTS = "Adults", _("Adults")
-    BOYS = "Boys", _("Boys")
-    DOMESTIC_VIOLENCE = "Domestic Violence (DV/IPV)", _("Domestic Violence (DV/IPV)")
-    FAMILIES = "Families", _("Families")
-    GIRLS = "Girls", _("Girls")
-    HIV_AND_AIDS = "HIV/AIDS", _("HIV/AIDS")
-    HUMAN_TRAFFICKING = "Human Trafficking", _("Human Trafficking")
-    LGBTQ = "LGBTQ", _("LGBTQ")
-    MEN = "Men", _("Men")
-    SENIORS = "Seniors (55+)", _("Seniors (55+)")
-    VETERANS = "Veterans", _("Veterans")
-    WOMEN = "Women", _("Women")
-    YOUTH = "Youth (TAY)", _("Youth (TAY)")
+    ADULTS = "adults", _("Adults")
+    BOYS = "boys", _("Boys")
+    CHILDREN = "children", _("Children")
+    DOMESTIC_VIOLENCE = "domestic_violence", _("Domestic Violence (DV/IPV)")
+    FAMILIES = "families", _("Families")
+    GIRLS = "girls", _("Girls")
+    HIV_AND_AIDS = "hiv_and_aids", _("HIV/AIDS")
+    HUMAN_TRAFFICKING = "human_trafficking", _("Human Trafficking")
+    LGBTQ = "lgbtq", _("LGBTQ")
+    MEN = "men", _("Men")
+    SENIORS = "seniors", _("Seniors (55+)")
+    VETERANS = "veterans", _("Veterans")
+    WOMEN = "women", _("Women")
+    YOUTH = "youth", _("Youth (TAY)")
+    JUSTICE_SYSTEM_EXITING = "justice_system_exiting", _(
+        "B7 Bridge Housing for Persons Exiting Justice System Institutions"
+    )
+    ENHANCED_BRIDGE_HOUSING_FOR_WOMEN = "enhanced_bridge_housing_women", _("Enhanced Bridge Housing for Women")
+    ENHANCED_BRIDGE_HOUSING_FOR_OLDER_ADULTS = "enhanced_bridge_housing_older_adults", _(
+        "Enhanced Bridge Housing for Older Adults"
+    )
```

Comment:
Should we care enough to put this in alphabetical order?

Reply:
we should